lib/cartopy/crs.py

```diff
@@ -2054,15 +2074,59 @@ def __init__(self, central_longitude=0.0, satellite_height=35785831,
             globe=globe,
             sweep_axis=sweep_axis)
 
+        # TODO: Let the globe return the semimajor axis always.
+        a = np.float(self.globe.semimajor_axis or WGS84_SEMIMAJOR_AXIS)
+        b = np.float(self.globe.semiminor_axis or a)
+        h = np.float(satellite_height)
+        max_x = h * np.arcsin(a / (a + h))
+        max_y = h * np.arcsin(b / (a + h))
+        self._set_bounds(max_x, max_y)
+
 
 class NearsidePerspective(_Satellite):
     """
     Perspective view looking directly down from above a point on the globe.
 
+    In this projection, the projected coordinates are x and y measured from
+    the origin of a plane tangent to the Earth directly below the perspective
+    point (e.g. a satellite).
+
     """
     def __init__(self, central_longitude=0.0, central_latitude=0.0,
                  satellite_height=35785831,
                  false_easting=0, false_northing=0, globe=None):
+        """
+        Parameters
+        ----------
+            central_longitude: float, optional
+                The central longitude. Defaults to 0.
+            central_latitude: float, optional
+                The central latitude. Defaults to 0.
+            satellite_height: float, optional
+                The height of the satellite. Defaults to 35785831 meters
+                (true geostationary orbit).
+            false_easting:
+                X offset from planar origin in metres. Defaults to 0.
+            false_northing:
+                Y offset from planar origin in metres. Defaults to 0.
+            globe: :class:`cartopy.crs.Globe`, optional
+                If omitted, a default globe is created.
+
+                .. note::
+                    This projection does not handle elliptical globes.
+
+        """
+        if globe is None:
+            globe = Globe(semimajor_axis=WGS84_SEMIMAJOR_AXIS, ellipse=None)
+
+        # TODO: Let the globe return the semimajor axis always.
+        a = globe.semimajor_axis or WGS84_SEMIMAJOR_AXIS
+        b = globe.semiminor_axis or a
+
+        if b != a or globe.ellipse is not None:
+            warnings.warn('The proj "nsper" projection does not handle '
+                          'elliptical globes.')
```

Comment:
So I've been thinking, maybe I should've moved this to a base class. Maybe change the default `Globe` creation based on an `__init__` parameter, or private class attribute. Something for another PR, though.

Reply:
yeah, I think there's a few more things we could hoist out of the leaf nodes in the tree of projections.

Reply:
Or more specifically - into a factory of Globe rather than the ``__init__`` of a baseclass.